app/apps/cases/views.py

```diff
@@ -115,13 +115,16 @@ def mock_cases(self, request):
     @action(detail=True, methods=["get"], url_path="tasks")
     def get_tasks(self, request, pk):
         case = self.get_object()
-        camunda_tasks = CamundaService().get_all_tasks_by_instance_id(case.camunda_id)
+        (camunda_tasks, response) = CamundaService().get_all_tasks_by_instance_id(
```

Comment:
Misschien een idee om hier de try catch te doen, en die in de CamundaService `_process_request` weg te laten. Dan kun je in de views de Django responses bepalen, in plaats van dat je het vanuit de service meegeeft. 

Je hebt nu ook een mix van Python response objecten en Django View Response objecten in je CamundaService code, dat wil je liever niet omdat je dan de view Response objecten aanpast om op de request response te lijken. (`response.ok = False`), dat is een beetje hacky